### ISSUE TITLE
[Feature Request] Changing visual subdivisions

### ISSUE THREAD

**Issue by Chickadee3141592** [HielkeMinecraft/OpenNoteBlockStudio#218]
**Is your feature request related to a problem? Please describe.**
Not really, it's more of a convenience thing. 

**Describe the enhancement you'd like**
I would like to be able to change the visual subdivisions. Right now, the 4/4 with 16th note subdivisions are unchangeable, and being able to change that to, say, 3/4 with triplet subdivisions or 4/4 32th note subdivisions, etc. would be nice.

**Comment by 4321ba** [comment]
I mean, you can change it from 2/4 to 8/4 in settings -> song properties, but I agree it's quite limited. (8/4 is kinda the same as 4/4 with 32th subdivisions.) But you can't change it to like 15/8 or something not to mention variable time signatures that change over time :D.

**Comment by Bentroen** [comment]
Yeah, right now the time divisions are very limited. While that setup works for most of the basic songs, I understand how more options would be useful. There are plans to make them more advanced, but likely only in v4.0. (It would also require another NBS format update to store the subdivisions.)
